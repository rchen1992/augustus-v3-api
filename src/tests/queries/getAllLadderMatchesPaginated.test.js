const mockLadderRepo = require('@tests/mockRepos/ladder');
const { testQuery } = require('@tests/utils');

testQuery({
    id: "Get all of a ladder's matches paginated with limit and offset",
    query: `
        {
            ladder(id: 1) {
                id
                matches(offset: 1, limit: 1) {
                    id
                    createdAt
                }
            }
        }
    `,
    variables: {},
    context: () => {
        return {
            repos: {
                ladder: mockLadderRepo,
            },
        };
    },
    expected: undefined,
});
