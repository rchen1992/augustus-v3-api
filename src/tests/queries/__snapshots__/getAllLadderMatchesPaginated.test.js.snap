// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`query: Get all of a ladder's matches paginated with limit and offset 1`] = `
Object {
  "data": Object {
    "ladder": Object {
      "id": "1",
      "matches": Array [
        Object {
          "createdAt": "2018-12-06 05:46:42",
          "id": "6",
        },
      ],
    },
  },
}
`;
